resources/views/articles/create.blade.php

```diff
@@ -0,0 +1,24 @@
+@extends('layouts.app')
+
+@section('content')
+
+<div class="bar-container">
+    <div class="bar-grid-create acc ripple">
+        <img class="bar-img" src="https://openclipart.org/image/800px/svg_to_png/320195/tropicalhouse-1831.png" alt="...">
+        <div class="bar-header">
+            <p class="bar-title">Hier können neue Artikel angelegt werden</p>
+        </div>
+    </div>
+
+    <form action="/articles" method="post" enctype="multipart/form-data">
+        <div class="main-accordion panel" style="display: block;">
```

Comment:
Inline styles are kinda bad. Use the Bootsrap class `block` instead.

Reply:
I'm not happy with that either, but it seems necessary for the accordion to be in open state when loading the page and still be able to toggle the state. Till now no bootstrap whatsoever in my app and I would like to keep it that way.

Seems like a good topic for tomorrows 1on1 :)